enterprise/server/api/api_server.go

```diff
@@ -339,6 +339,11 @@ func (s *APIServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *APIServer) DeleteFile(ctx context.Context, request *apipb.DeleteFileRequest) (*apipb.DeleteFileResponse, error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
instead of using panic (we almost never use it in server code) you can use:
```
return nil, status.UnimplementedError("not yet implemented")
```

Reply:
Done